src/devol.cpp

```diff
@@ -170,9 +170,10 @@ void devol(double VTR, double f_weight, double f_cross, int i_bs_flag,
                 break;
             }
             case 6: {                           // ---DE/current-to-p-best/1 (JADE)-----------------------------------
-                int i_pbest = sortIndex[static_cast<int>(::unif_rand() * p_NP)]; // select from [0, 1, 2, ..., (pNP-1)] 
+                int i_pbest; // select from [0, 1, 2, ..., (pNP-1)] 
```

Comment:
Technically we don't that line.  The (re-)declaration on line 176 shadows it.  Or am I missing something?

Reply:
My mistake - this is how I meant to have it. Though if you want it just declared at line 176 for each iteration? You know better than I!!